docs/operator-manual/disaster_recovery.md

```diff
@@ -26,3 +26,5 @@ docker run -i -v ~/.kube:/home/argocd/.kube --rm argoproj/argocd:$VERSION argocd
 
 !!! note
     If you are running Argo CD on a namespace different than default remember to pass the namespace parameter (-n <namespace>). 'argocd admin export' will not fail if you run it in the wrong namespace.
+    example: kubectl config set-context --current --namespace=argocd
```

Comment:
This is an example of how to avoid using the `-n` parameter. Can you rephrase to show that this is an _alternative_ to the `-n` parameter?